Source/OxyPlot/Series/FinancialSeries/HighLowSeries.cs

```diff
@@ -97,13 +88,7 @@ public OxyColor ActualColor
         /// Gets the items of the series.
         /// </summary>
         /// <value>The items.</value>
-        public List<HighLowItem> Items
-        {
-            get
-            {
-                return this.items;
-            }
-        }
+        public List<HighLowItem> Items { get; } = new List<HighLowItem>();
```

Comment:
Weird it didn't `new()` this; am I missing something?

Reply:
I changed this to new(), and also some other places I found. Probably did not get all of them, but shouldn't matter too much I would think.